### DISCUSSION TITLE
Add To Do list directly in report issue page

### DISCUSSION THREAD

**Discussion by priocomdevops** [andrzejkupczyk/mantisbt-todolists#46] [Q&A] [answered]
I'd like to include a To Do list directly on the 'Report Issue' Page.

Currently, to add a To-Do-list,  it is a two-step process whereby one first creates an issue, then has to navigate back to 'View Issue Details' page to add the To Do List.

Is this possible?

**Comment by andrzejkupczyk** [comment, marked as answer]
Hey @priocomdevops, while this is a good idea (thanks!), it will be quite difficult to implement. Creating a To-Do list requires an issue to exist first, which is why the current process involves creating an issue before adding a To-Do list. I hope this explanation makes sense to you. Please let me know if you have any further questions or suggestions.
